src/wines/dtos/update-wine.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty, PickType } from '@nestjs/swagger';
+import { Wine } from '../entities/wine.entity';
+
+export class AddStoreToWineDto extends PickType(Wine, ['stores'] as const) {
```

Comment:
Isn't this class completely irrelevant now? Remove or add comment to explain why it should exist without usage

Reply:
you are right, deleted